### DISCUSSION TITLE
Simple little details

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#528] [General]
As the title suggests, simple details that's somewhat unnecessary and no one will notice but maybe will be worth the change.

- Increase retweet button's saturation, I really sometimes don't know if i retweeted or not

- some widgets on timeline needs a touching animation (when pressing and holding on various things)

- Make copying the tweet by pressing and holding on the text itself. 

- Make the mentions tab have two or more different categories in itself or make a toggle in settings to make it either general notifications or custom. (By categories i mean interactions such as likes, retweets and follows etc, and another one for mentions) 
Experiment as you like and implement the best one

- Put secret Easter eggs or fun effects :)

- put an option to use the user's device font, I have a font with iOS emojis and I noticed a lot of apps making it useless as it uses a font of their choice. 

- Photo editing, cropping and adding text and minor features like that.

- Reply Count?

- Private account symbol (🔒)

- Confirmation dialog when unfollowing Private accounts.

And I'm open to discuss these or anything else regarding "minimal features"

**Comment by robertodoering** [comment]
Thanks for the suggestions!
Some of is already planned, including better support for private accounts (the confirmation dialog is a great idea).

Which theme are you using where the retweeted button color is hard to see?

**Reply by FerasK0** [reply]
I didn't buy the pro yet and i don't know if retweet color is customizable.
But the crow and harpy is where the problem is. 

<img width=250 src="https://user-images.githubusercontent.com/83197825/149813590-06648904-12ae-438f-94a8-08041461e4ae.jpg">
<img width=250 src="https://user-images.githubusercontent.com/83197825/149813604-c2fa8cdf-4391-4d78-a0bb-b88d09a93f45.jpg">
<img width=250 src="https://user-images.githubusercontent.com/83197825/149813620-d895e96f-0e8b-4d35-8f94-0d046d481ead.jpg">

Also i got an idea to support the color blind with some high contrast mode or something.

**Reply by robertodoering** [reply]
You are right it definitely lacks contrast for the dark themes.
I will update them, thanks.
